### DISCUSSION TITLE
Windows 10, only DNS traffic is routed to VPN

### DISCUSSION THREAD

**Discussion by rnrib** [strongswan/strongswan#535] [Q&A] [answered]
Hello all,

Maybe somebody has an idea or could give me a hint on how to continue troubleshooting this.

I've configured strongswan on AWS and I can connect successfully from my Android devices except for my Windows 7 desktop.
The VPN (IKEv2 with machine certs) is successfully established, but only DNS traffic is going to the tunnel.
On the AWS server in tcpdump I see only DNS requests and replies which tells, that routing and iptables are fine.
But I don't see any other traffic coming from the Windows machine. 

Nslookup works great with different DNS servers, but that's about that. 

I've tried:
1) I've configured it on the other Windows machine, and it works fine!
2) Upgraded Windows 7 to Windows 10
3) Performed clean boot
4) Deleted network drivers and reinstalled them
5) Disabled antivirus, firewall

Since DNS is working with different public DNS, I assume that it's neither server problem, nor routing.  Something blocks traffic locally but I cannot understand what it could be.

I understand, that since it works everywhere except my machine, probably clean Windows 10 install will help. But that's something I would like to avoid, at least for now

**Comment by Thermi** [comment]
Take a look at [this article](https://wiki.strongswan.org/projects/strongswan/wiki/WindowsClients#Split-routing-on-Windows-10-and-Windows-10-Mobile), specifically the section "Split routing on Windows 10 and Windows 10 Mobile"  on the wiki. It's probably the solution you're looking for

**Reply by rnrib** [reply]
Thermi, thank you for your reply,

but this is not a routing issue. 
As I told, DNS-traffic to/from **any** DNS server is routed to/from the tunnel. This is confirmed by:
1) I see the DNS-traffic on the VPN-server with tcpdump
2) Nslookup on the Windows machine works perfectly with different DNS servers
3) Other Windows machines work fine with my VPN-server.

Being a network engineer for some while, the routing stuff was the first thing I have checked, but unfortunately, I lack Windows expertise and I haven't found anything similar in google.

**Comment by rnrib** [comment, marked as answer]
Eventually, I managed to find an issue,

A couple of years ago, I worked in a Russian bank, And the Russian government allows banks to use only approved sets of ciphers for encryption. So, for a remote connection standard IPSec is not an option. That time I used some Russian hardware and software for a VPN, and there was a weird driver called Amicon NDIS 6.20 Filter Driver (although I removed the software long ago). 
This driver was installed and could be seen in the properties of network adapters along with TCP/IP for IPv4, IPv6 etc. Even though it was unchecked, it somehow affected my VPN. Once I removed it completely, everything started to work.

So, some archeological issue:)
